### PR TITLE
Render avg rating by filling in stars

### PR DESCRIPTION
**Pull request by shawnsull3**
- Create a helper function called calcAvgRating based on the review meta data

- Created a stars react component that will display the avg rating by filling in stars.

- @thomHayner @jwhooperjr @morahan be sure to perform an npm install after pull from master. I added the react-star-ratings dependency

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ class App extends Component {
   render() {
     return (
       <div className="App">
-        <div className="container">
+        <div className="container-fluid">
```
Comment (shawnsull3) [inline src/App.jsx:27]: Attempting to get the rows to completely fill the window

File: src/data/types/types.jsx [hunk 2]
```diff
@@ -2,12 +2,8 @@
 export const CHANGE_PRODUCT_ID = 'CHANGE_PRODUCT_ID';
 export const GET_REVIEW_META_DATA = 'GET_REVIEW_META_DATA';
 export const GET_REVIEW_META_DATA_SUCCESS = 'GET_REVIEW_META_DATA_SUCCESS';
-export const GET_REVIEW_META_DATA_FAILURE = 'GET_REVIEW_META_DATA_FAILURE';
```
Comment (shawnsull3) [inline src/data/types/types.jsx:5]: Not using this type for actions/reducers, so I removed it

**Comment by shawnsull3** [comment]
The stars are functional, currently just reading the data from product_id: 1. More styling is needed, but I need to figure out bootstrap first.

**Review by morahan — APPROVED** [review]
minor updates to existing files.  Addition of new shared files: HelperFunctions.js (average star rating), stars.jsx. delete stars file in product overview

**Pull request merged**